SETUP/API.md

```diff
@@ -5,14 +5,18 @@ such as projects, word lists, and statistics.
 
 ## Configuring
 
-To enable the API, update the following API-related values in your
-`configuration.sh` file and re-configuring your site.
+The API is enabled by default because some UI features, like the Page Browser,
+require it.
 
-* `API_ENABLE` - if set to `TRUE` the API is enabled.
+While disabling the API will break these features, you can still do so by
+updating the `API_ENABLE` setting in your `configuration.sh` file and
+re-configuring your site.
 
-It is also highly recommended that you update your web browser to rewrite
-requests for `/api/` to `api/index.php?url=`. This will enable access to
-URLs like:
+* `API_ENABLE` - if set to `FALSE` the API is disabled.
+
+While not required for the DP UI, for external users it is also highly
+recommended that you update your web browser to rewrite requests for `/api/`
+to `api/index.php?url=`. This will enable access to URLs like:
```

Comment:
I find "web browser" confusing -- is this instruction meant for individual users, or for the installer of the code.

Other than that confusion, looks good.

Reply:
That should say `web server` - thanks for catching that. I'll get it fixed.